src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -15,24 +14,20 @@ import {
   SplitItem,
   Title
 } from '@patternfly/react-core';
+import { Table, TableHeader, TableBody, SortByDirection } from '@patternfly/react-table';
 import {
-  Table,
-  TableHeader,
-  TableBody,
-  SortByDirection,
-  sortable,
-  cellWidth,
-  expandable
-} from '@patternfly/react-table';
+  BooleanDictionary,
+  countManifests,
+  getTableHeaders,
+  getManifestName,
+  getRowsWithAllocationDetails
+} from './satelliteManifestPanelUtils';
```

Comment:
I followed up on Slack, but as mentioned, I think at the very least if we are going to reconfigure this component it may be a bigger lift than what is feasible for this story, as I'm finding it challenging, and it may just not be possible, to find a clean way to partition this so that it can be tested exclusively via props / snapshots. Maybe we should leave the utils for now and think about refactoring this component more later on.